### PR TITLE
fix: adapt to topos CLI changes

### PR DESCRIPTION
**Pull request by JDawg287**
# Description

This PR adapts to the major changes in the `topos` CLI tool.
See changes in [#352](https://github.com/topos-protocol/topos/pull/352/files).

Fixes TP-768

## Additions and Changes

- Removed not used commands anymore, `sequencer`, `tce`
- Added static default config files for the boot node
- Changed command to run the boot node

## Breaking changes

REMOVED UNUSED COMMANDS FOR THE CLI TOOL

## PR Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added or updated tests that comprehensively prove my change is effective or that my feature works

### CODE REVIEW

**Review by dvdplm — COMMENTED** [review]
Unsure how to best review this. Have we tested this in devnet yet?

**Comment by JDawg287** [comment]
@dvdplm Since setting up the local infra requires starting a docker-compose setup from various repositories, the easiest way is to set up the GitHub refs or Docker tags manually in the [e2e tests](https://github.com/topos-protocol/e2e-tests/actions/workflows/frontend:erc20-messaging.yml) and run the tests.
If all goes well, the tests would pass and we can consider all components functioning properly.
The tests are failing for now, so looking at it at the moment.

File: subnet-incal.yml [hunk 1]
```diff
@@ -153,17 +157,15 @@ services:
     entrypoint: ""
     command: bash -c "
       source /contracts/.env &&
-      export SUBNET_CONTRACT_ADDRESS=$(printenv TOPOS_CORE_PROXY_CONTRACT_ADDRESS) &&
-      init.sh boot sequencer run"
+      echo Topos core contract address=$(printenv TOPOS_CORE_PROXY_CONTRACT_ADDRESS) &&
```
Comment (sebastiendan) [inline subnet-incal.yml:160]: Is this used anywhere?

Comment (atanmarko) [reply subnet-incal.yml]: Yes to check if configuration file has correct address. 
Due to changes in topos cli we have regression not to be able to set contract address from the environment. We will fix that soon

**Review by atanmarko — APPROVED** [review]

File: .github/workflows/test:e2e.yml [hunk 2]
```diff
@@ -17,4 +17,4 @@ jobs:
           workflow_file_name: frontend:erc20-messaging.yml
           ref: main
           wait_interval: 60
-          client_payload: '{ "local-erc20-messaging-infra-ref": "${{ github.head_ref }}" }'
+          client_payload: '{ "local-erc20-messaging-infra-ref": "${{ github.head_ref }}", "topos-docker-tag": "main" }'
```
Comment (sebastiendan) [inline .github/workflows/test:e2e.yml:20]: @atanmarko This should not be set explicitly

**Pull request merged**